app/components/certificate/certificate-request.tsx

```diff
@@ -0,0 +1,41 @@
+import { Flex, Heading, Text, HStack, Button } from '@chakra-ui/react';
+
+import Description from './description';
+
+interface CertificateRequestViewProps {
+  subject: string;
+  validFrom: Date;
+  validTo: Date;
+  handleSubmit: () => void;
```

Comment:
To be consistent with how React does this, you'd name this `onRequest`.  Event handlers use the `on*` prefix.

Reply:
Ohh I'll remember that